.gitignore

```diff
@@ -3,3 +3,15 @@ bin/midenc
 bin/litcheck
 .crates.toml
 .crates2.json
+
+# Docs platform ignores
```

Comment:
*Note:* Our docs build in the `target` directory, which is already ignored, and our `docs` directory has a dedicated `.gitignore` for things within that directory, so most of these directives will never be matched.

That said, I'm fine with adding these here anyway (mostly the IDE-specific directives, since we should have those anyway), just in case someone accidentally builds the docs in the project root locally - but just wanted to clarify where you can expect these things to actually be output in this repo.

Reply:
Yes, whenever I spin it up locally and make changes, I'm prompted to push a whole load of unnecessary stuff ... so that's why I added these like we have in the Miden base repo.